boards/BMS/Inc/App/App_CellVoltages.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include <stdint.h>
+
+/**
+ * Initialize a set of functions used to calculate voltages.
+ * @param get_cell_voltages A pointer to a 2D array containing raw cell voltages
+ * measured by a daisy chain of cell monitoring chips.
+ * @param num_of_cells_per_segment The number of cell voltage measurements per
+ * segment.
+ * @note The raw cell voltages are read back in 100µV. Divide the raw voltages
+ * by 10000 to compute the voltage in V.
```

Comment:
👀

Reply:
Noice do you think a #define would be nice as well for the conversion factor or nah?

Reply:
Hmm yeah good idea

Reply:
I added a define here `V_PER_100V`